xslt-ballerina/Package.md

```diff
@@ -1,5 +1,15 @@
 ## Package Overview
+The XSLT library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
+</a> language.
 
 This package provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.
 
-For information on the operations, which you can perform with this module, see the below **Functions**. For examples on the usage of the operations, see the [XSLT Transformation Example](https://ballerina.io/learn/by-example/xslt-transformation.html).
\ No newline at end of file
+### Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```

Comment:
```suggestion
To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina Standard Library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/138/commits/18cfd06cc69a76b0cb2a6d6bff3e7c847b6c8304